frontend/routes/manage/questionnaire/Message.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div>
+        <h1>{{ $t("pages.manage.message.title") }}</h1>
+        <hr>
+        <TextInput :placeholder="$t('pages.manage.message.subject')"></TextInput>
+        <Dropdown>
+            <option selected>Bee</option>
+            <option>2</option>
+        </Dropdown>
+        <br>
```

Comment:
I'd like to avoid using `<br/>`s instead of CSS for vertical spacing. I think if you wrap the dropdown, textarea, and button in a `<div class="row"><div class="col-12">`, that should add the spacing you're looking for, but I might be misremembering the CSS.